### ISSUE TITLE
Empty References

### ISSUE THREAD

**Issue by christmasunicor** [OpenGreekAndLatin/First1KGreek#2086]
@lcerrato As I understand it, the latest version of [hooktest](https://github.com/Capitains/HookTest/releases) automatically checks for empty references. On one of my recent builds in Travis, I indeed receive the error "empty references" for about 4 files. (The log is [here, errors at lines 441ff.] (https://pastebin.com/GMcKD3Sq). Trouble is, I don't know what an "empty reference" is: could you elaborate? The documentation for HookTest is unfortunately sparse, and I haven't had any luck with the TEI guidelines either. :cold_sweat:

**Comment by sonofmun** [comment]
Hi @christmasunicor, @lcerrato is exactly right here. This was just implemented last Friday in HookTest and the message should tell you at what level your empty reference is. I.e., if it says that you have 1 empty reference(s) at level 1, that means that if your citation scheme is book, chapter, section, that one of your `<div type="textpart" subtype="book">` elements has an empty `@n` attribute. This is something that is unique to HookTest and CapiTainS, since TEI is agnostic about even whether you have a citation scheme.

**Comment by byeats** [comment]
Thanks for both of your comments: they cleared everything up. @lcerrato, I accidentally commented using my personal account, so @christmasunicor and I are the same person. See my pull request for these 4 fixed files. I know that we usually operate by "1 pull request, 1 file," but these files all correspond to this issue, so I ventured to keep them in one pull request. I can separate them, if necessary.

**Comment by lcerrato** [comment]
@byeats 
Yes, one pull request per file allows for including the file name in the pull request. Our preferred format for an issue name or pull request name is  `(file name) issue description`so when multiple files are included, that doesn't work as well.
As noted in the pull request, this is fine for now — no need to break up the files.
